tests/test_cli.py

```diff
@@ -17,29 +45,205 @@ def test_help():
+        with caplog.at_level("INFO"):
+            assert "Job submitted successfully" in caplog.text
+
+
 def test_cellranger_arc_count():
     runner = CliRunner()
     result = runner.invoke(cli, ["alignment", "cellranger-arc-count", "--help"])
     assert result.exit_code == 0
     assert "Show this message and exit" in result.output
 
 
-## Tests for irods commandds
+def test_cellranger_arc_count_invalid_libraries(caplog):
+    runner = CliRunner()
+    result = runner.invoke(
+        cli,
+        ["alignment", "cellranger-arc-count", "--libraries", "test.csv"],
+        catch_exceptions=False,
+    )
+
+    assert result.exit_code != 0  # Expected to fail
+    with caplog.at_level("ERROR"):
+        assert "Invalid value for '--libraries': Path" in result.output
+
+
+def test_cellranger_arc_count_valid_sample(caplog):
+    # Make mock of subprocess.run to simulate a successful 'process'
+    with patch("subprocess.run") as mock_run:
+        mock_process_arc = MagicMock()
+        mock_process_arc.stdout = "Job submitted successfully"
+        mock_run.return_value = mock_process_arc
+
+        runner = CliRunner()
+        result = runner.invoke(
+            cli,
+            ["alignment", "cellranger-arc-count", "--libraries", libraries_path],
+            catch_exceptions=False,
+        )
+
+        assert result.exit_code != 0
+        with caplog.at_level("ERROR"):
+            assert "No valid libraries files found. Exiting" in caplog.text
+
+
+def test_cellranger_vdj():
+    runner = CliRunner()
+    result = runner.invoke(cli, ["alignment", "cellranger-vdj", "--help"])
+    assert result.exit_code == 0
+    assert "Show this message and exit" in result.output
+
+
+def test_cellranger_vdj_invalid_sample(caplog):
+    runner = CliRunner()
+    result = runner.invoke(
+        cli,
+        ["alignment", "cellranger-vdj", "--sample", "fail-sample"],
+        catch_exceptions=False,
+    )
+
+    assert result.exit_code != 0  # Expected to fail
+    with caplog.at_level("ERROR"):
+        assert "No valid samples found. Exiting" in caplog.text
+
+
+def test_cellranger_vdj_valid_sample(caplog):
+    # Make mock of subprocess.run to simulate a successful 'process'
+    with patch("subprocess.run") as mock_run:
+        mock_process = MagicMock()
+        mock_process.stdout = "Job submitted successfully"
+        mock_run.return_value = mock_process
+
+        runner = CliRunner()
+        result = runner.invoke(
+            cli,
+            ["alignment", "cellranger-vdj", "--sample", "sample_test"],
+            catch_exceptions=False,
+        )
+
+        assert result.exit_code == 0
+        assert "Job submitted successfully" in caplog.text
+
+
 def test_imeta_report():
     runner = CliRunner()
     result = runner.invoke(cli, ["irods", "imeta-report", "--help"])
     assert result.exit_code == 0
     assert "Show this message and exit" in result.output
 
 
+#####   FAILING
```

Comment:
If you don't want to merge this yet because you're still working on it, please stash in a new branch or Issue for future work.